dev-tools/src/main/resources/plugin-metadata/plugin-descriptor.properties

```diff
@@ -34,6 +34,8 @@
 description=${project.description}
 #
 # 'version': plugin's version
+# version string must be a sequence of nonnegative decimal integers
+# separated by "."'s and may have leading zeros
 version=${project.version}
```

Comment:
I dont think this is correct. I think you mean to document the java version (not the one of the plugin). I think we should also just say its the java.specification.version?